.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+All pull requests must have an associated issue in issue tracker. If there
```

Comment:
"in THE issue tracker", or does this just sound strange to me because I am Dutch? :)